source/apiVolontaria/location/tests/tests_model_Address.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework.test import APITestCase
+
+from ..models import Address, Country, StateProvince
+
+
+class AddressTests(APITestCase):
+
+    def setUp(self):
+        self.random_country = Country.objects.create(name="random country")
+        self.random_state_province = StateProvince.objects.create(
+            name="random state",
+            country=self.random_country,
+        )
+
```

Comment:
Need an other test to check if your `clean()` method works (conflicts between `state_province.country` and `country`)

Reply:
Right!